ipalib/misc.py

```diff
@@ -26,26 +26,31 @@ class env(LocalOrRemote):
     )
 
     takes_options = LocalOrRemote.takes_options + (
-        Flag('all',
+        Flag(
```

Comment:
Wouldn't it be better to go like this?
```python
Flag('all',
     cli_name='all',
     ...
     )
```
Otherwise it looks a bit ridiculous as you're saving about 1-2 spaces here :)